Code/Framework/AzNetworking/AzNetworking/AutoGen/AutoPacketDispatcher_Inline.jinja

```diff
@@ -6,16 +6,25 @@ namespace {{ xml.attrib['Name'] }}
     {
         switch (aznumeric_cast<int32_t>(packetHeader.GetPacketType()))
         {
+{% set packet_ns = namespace(handshake=false) %}
+{%      for Packet in xml.iter('Packet') %}
+{%          if ('HandshakePacket' in Packet.attrib) and (Packet.attrib['HandshakePacket'] == 'true') %}
```

Comment:
Since there are so many ways to express a true value, you might want to use the |booleanTrue filter instead for these checks. It will offer less surprises if someone writes "True" or "true " or "1" or " True", etc..

Reply:
Ah good to know, I'll give that a try.